libs/cimit-service/src/main/java/uk/gov/di/ipv/core/library/service/CiMitService.java

```diff
@@ -437,9 +284,13 @@ private HttpResponse<String> sendHttpRequest(HttpRequest cimitHttpRequest)
             LOGGER.info(LogHelper.buildLogMessage("Sending HTTP request to CiMit."));
             return httpClient.send(cimitHttpRequest, HttpResponse.BodyHandlers.ofString());
         } catch (InterruptedException e) {
+            LOGGER.error(
+                    LogHelper.buildErrorMessage(
+                            "Client interrupted sending http request to CiMit", e));
```

Comment:
I know we have it inconsistent throughout our code, but can we try to align on `CIMIT` as a stylisation going forward? (Likewise `SPOT` over `SPoT`)

I asked Chris C and he agrees - he thinks `CiMit` might have been a Slade-ism.

Reply:
ha - yep. Good to have an offical steer.